### PR TITLE
encapsulte util functions in a util object for clarity

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: src/WorldGenerator.js [hunk 1]
```diff
@@ -9,17 +9,17 @@ WorldGenerator.prototype = {
...
     planetData.generateMap();
-    game.cache.addBitmapData('randomTex', randomTexture());
+    game.cache.addBitmapData('randomTex', utils.randomTexture());
     /* generate a  bitmap to use as a fancy background image in the level */
     /* image is a quarter of the screen size because we do not need high-res */
     var bmd = game.make.bitmapData(game.width * 0.5, game.height * 0.5);
     var h, c, s;
     for (var x = 0; x < bmd.width; x++) {
       for (var y = 0; y < bmd.height; y++) {
         h = roundTo(noise.fbm2(x * 0.01, y * 0.01, 8) * ((y + 1) / bmd.height), 0.15 * ((y + 1) / bmd.height));
```
Comment (clayjohn) [inline src/WorldGenerator.js]: roundTo needs a util in front of it

**Comment by CalebJohn** [comment]
This looks very very good, unfortunately I won't have the opportunity to test this until tomorrow afternoon. It should be ready to merge as soon as I get the chance to test.

**Comment by clayjohn** [comment]
Looks good to me now with the last commit.

**Comment by CalebJohn** [comment]
Sorry for the delay getting to this one, it was a simple change but I wanted to make sure I gave it a proper check.

**Pull request merged**